wiki/Community/Forum/Modding_Queues/en.md

```diff
@@ -0,0 +1,27 @@
+---
+stub: true
+---
+
+# Modding Queues
+
+<!-- TODO: this needs to be linked from other related articles, for example, "how to get mods" -->
+
+[**Modding Queues**](https://osu.ppy.sh/community/forums/60) is a forum dedicated to [beatmap review](/wiki/Modding) requests, or mods. It consists of forum threads, or *queues* belonging to different modders and mappers. Every queue has individual requirements and limits on obtaining the mods, which must be respected by its visitors in order to receive a mod.
+
+Since 2011-11-30,<!-- TODO: add https://github.com/ppy/osu-web/issues/8256 as a footnote when they're supported --> each game mode has its own subforum for queues, with osu! taking up main section of the forum:
```

Comment:
that's table format (also wrong decade rofl)

```suggestion
Since November 30, 2021,<!-- TODO: add https://github.com/ppy/osu-web/issues/8256 as a footnote when they're supported --> each game mode has its own subforum for queues, with osu! taking up the main section of the forum:
```

but isn't this more fit for the history articles anyway?

```suggestion
Each game mode has its own subforum for queues, with osu! taking up the main section of the forum:
```

Reply:
maybe, I felt like it was important at the moment of editing. feel free to take it out